server/src/modules/market-making/strategy/strategy.service.ts

```
@@ -132,8 +146,13 @@ export class StrategyService
           errorTrace,
         );
       } finally {
-        session.nextRunAtMs += session.cadenceMs;
-        this.sessions.set(session.strategyKey, session);
+        if (!this.sessions.has(session.strategyKey)) {
+          continue;
```

Biome (2.4.4)

[error] Unsafe usage of 'continue'.

(lint/correctness/noUnsafeFinally)